### PR TITLE
Fixes #19485: Sync package modules from 3.18.0

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19485

### CODE REVIEW

File: tree/10_ncf_internals/modules/packages/nimclient [hunk 1]
```diff
@@ -63,7 +63,7 @@ list_updates () {
     # This function should return the list of package updates that are available
     # NAME=
     # VERSION=
-    # ARCHITECTURE=
+    # ARCHITECUTRE=
```
Comment (Fdall) [inline tree/10_ncf_internals/modules/packages/nimclient:66]: Not sure about this one

**Review by Fdall — CHANGES_REQUESTED** [review]

**Comment by amousset** [comment]
PR updated with a new commit

**Review by Fdall — APPROVED** [review]

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**